src/main/java/uk/gov/hmcts/dts/fact/services/admin/AdminCourtAreasOfLawService.java

```diff
@@ -0,0 +1,39 @@
+package uk.gov.hmcts.dts.fact.services.admin;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.dts.fact.exception.NotFoundException;
+import uk.gov.hmcts.dts.fact.model.admin.AreaOfLaw;
+import uk.gov.hmcts.dts.fact.repositories.CourtRepository;
+
+import java.util.List;
+
+import static java.util.stream.Collectors.toList;
+
+@Service
+public class AdminCourtAreasOfLawService {
+
+    private final CourtRepository courtRepository;
+
+
+    @Autowired
+    public AdminCourtAreasOfLawService(final CourtRepository courtRepository) {
+        this.courtRepository = courtRepository;
+    }
+
+    public List<AreaOfLaw> getCourtAreasOfLawBySlug(final String slug) {
+
+        return courtRepository.findBySlug(slug)
+            .map(c -> c.getAreasOfLaw()
+            .stream()
+            .map(AreaOfLaw::new)
+            .collect(toList()))
+            .orElseThrow(() -> new NotFoundException(slug));
```

Comment:
Think there is slight indentation issue here. The line '.orElseThrow is outside of the brackets for the first '.map'. These  2 line are ok to be indented the same away. The lines between these 2 should be further indented as they are within the brackets for the first '.map'

Reply:
couldn't work out what indentation issue as no check style or pmd rules violated

Reply:
If you move the .stream() back to the end of the previous line and then click enter to move it down again, you can see that it is further indented compared to the line above. You can see an example for this on getCourtContactsBySlug(). This is because the .stream(), second .map() and .collect() and are within the first .map()

Reply:
have amended with the correct indentation